chart/otomi/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: A Helm chart for installing otomi in Kubernetes
 home: https://otomi.io/
 icon: https://otomi.io/img/otomi-logo.svg
 type: application
-version: 0.3.CHART_PATCH_PLACEHOLDER
+version: 0.3.0-CHART-PATCH-PLACEHOLDER
```

Comment:
Previous is invalid semver.

Reply:
But you  can't just replace it without modifying the code that depends on it. Did you not look at how `CHART-PATCH-PLACEHOLDER` is used in the code?

Reply:
I'll update the dependency.

I still think we should have valid semver, even for a placeholder, because in my opinion there is a valid use case for `helm apply` this chart locally, and `helm lint` will throw an error now.